dev_local.sh

```diff
@@ -35,24 +35,30 @@ if [ $# -ne 0 ] ; then
     exit 2
 fi
 
-echo -n "Check telepresence version "
-if [ ! telepresence version | grep -q "Client: v2" ]
+TELEPRESENCE_VERSION="v2"
+if telepresence version | grep "Client: $TELEPRESENCE_VERSION" > /dev/null
```

Comment:
This will also match `Client: v2000-01-01`, maybe use more exact match?

Reply:
:+1: